site-src/geps/gep-709.md

```diff
@@ -32,17 +32,17 @@ emerge.
 
 To ensure that Gateway API is able to safely provide this functionality, we need
 to enforce a handshake mechanism that requires resources in both namespaces to
-agree to this reference. To accomplish that, a new ReferencePolicy resource
+agree to this reference. To accomplish that, a new ReferenceGrant resource
 should be introduced.
 
-![Reference Policy](images/709-referencepolicy.png)
+![Reference Policy](images/709-referencegrant.png)
```

Comment:
I think this image source needs to be renamed too.

Reply:
Updated, thanks!